cfgov/jinja2/v1/landing-page/index.html

```diff
@@ -1,23 +1,29 @@
-
 {% extends 'layout-2-1-bleedbar.html' %}
 
 {% import 'molecules/related-posts.html' as related_posts with context %}
 
 {% if not page %}
-      {% set doc = get_document('posts', 'four-years-working-for-you') %}
-      {% do global_dict.update(
-        {
+    {% set doc = get_document('posts', 'four-years-working-for-you') %}
+    {% do global_dict.update({
             'page':  {
                 'seo_title': 'This is a prototype page for testing landing page layouts.',
-                'related_posts': {
-                    'posts': more_like_this(doc, search_types='posts', search_size=3),
-                    }
+                'related_posts': global_dict.related_posts_function
+            },
+            'related_posts_settings': {
+                'value': {
+                    'limit': 3,
```

Comment:
We've been indenting the values after the keys so that it's easier to scan visually. See https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/jinja2/v1/_includes/molecules/form-field-with-button.html#L13-L24